src/main/steps/index.ts

```diff
@@ -63,7 +63,13 @@ const getNextIncompleteStep = (
 export const getNextIncompleteStepUrl = (req: AppRequest): string => {
   const { queryString } = getPathAndQueryString(req);
   const sequence = getUserSequence(req);
-  const url = getNextIncompleteStep(req.session.userCase, sequence[0], sequence, true);
+  const sequenceIndex =
+    !req.session.isApplicant2 &&
+    (req.session.userCase.state === State.ConditionalOrderDrafted ||
```

Comment:
might be cleaner to do [State.ConditionalOrderDrafted, State.ConditionalOrderPending].includes(req.session.userCase.state)